cmd/preupgradechecks/Dockerfile.fission-preupgradechecks

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine as godep
+FROM golang:1.14-alpine as godep
```

Comment:
Let's make 1.14 update a separate PR, please.

Reply:
It's actually a different PR. Just built this on top of it. We just need to merge that one first https://github.com/fission/fission/pull/1976